frontend/components/Home.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+import { Alert, Grid, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import Header from './Header';
+import ToolCard from './ToolCard';
+import { Tool } from '../interfaces';
+import '../css/Home.css';
+
+const Root = styled('div')(({ theme }) => ({
+  margin: theme.spacing(2)
+}));
+
+const MainContainer = styled('div')(({ theme }) => ({
+  margin: theme.spacing(3)
+}));
+
+const filterTools = (tools: Tool[], filter: string): Tool[] => {
+  const filterLower = filter.toLowerCase();
+  return tools.filter(
+    (tool) => (
+      tool.name.toLowerCase().includes(filterLower) ||
+      tool.short_description.toLowerCase().includes(filterLower))
+  );
+};
+
+function Home() {
+  const [tools, setTools] = useState<null | Tool[]>(null);
+  const [searchFilter, setSearchFilter] = useState('');
+
+  useEffect(() => {
+    const fetchToolData = async () => {
+      const url = '/api/lti_tools/';
+      const response = await fetch(url);
+      const data: Tool[] = await response.json();
+      // sort data alphabetically by name
+      data.sort((a, b) => (a.name.toLowerCase() > b.name.toLowerCase())
+        ? 1
+        : ((b.name.toLowerCase() > a.name.toLowerCase()) ? -1 : 0)
+      );
+      setTools(data);
+    };
+    fetchToolData();
+  }, []);
```

Comment:
This was tidied up in the process of converting; we didn't need a separate `useEffect` or state variable for handling the filter. The component will re-render and filter the tools down when `searchFilter` changes. See line 50.

Reply:
I think I resolved #168 here. I'm mostly of the opinion that `console.log` should be used sparingly or not at all, but `console.error` is fine to use when needed. I use `console.log` a lot in debugging though.

Reply:
Yeah, maybe there are some FE strategies for logging that I haven't explored.